tests/e2e/runner/.dockerignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-generated
\ No newline at end of file
+generated
+.env
```

Comment:
Are we ever running these tests with non-local env vars? If not maybe we want a committed test env file?

Reply:
I can't think of a reason why we would be running these tests with non-local env vars (since they are so tightly coupled with the seed subgraph config). Part of this was just to update these to be consistent with our .gitignore, so we don't have different behavior in docker on different machines.

I was playing around with running codegen locally, which requires a populated environment var, so @yoshiwarab do you agree that it seems preferable to commit a .env file instead of defining the value in the docker-compose yaml?

Reply:
Ah forgot they were already in docker compose,   let's leave this then!

Reply:
okay - will leave as-is 👍